Source/Foundation/Extensions/DateExtensions.swift

```diff
@@ -35,48 +35,45 @@ public extension Date {
         return timeIntervalSince1970 * 1_000.0
     }
 
-    var day: String {
-        return formatWith(dateFormat: "d")
+    var formattedYear: String {
+        return format(dateFormat: "yyyy")
     }
 
-    var monthSpanishNameAndYear: String {
-        return formatWith(dateFormat: "MMMM yyyy")
+    var formattedShortYear: String {
+        return format(dateFormat: "yy")
     }
 
-    var monthSpanishName: String {
-        return formatWith(dateFormat: "MMMM")
-    }
-
-    var shortMonthSpanishName: String {
-        return formatWith(dateFormat: "MMM")
-    }
-
-    var year: String {
-        return formatWith(dateFormat: "yyyy")
+    var formattedISO8601Date: String {
+        return DateFormatter.iso8601DateFormatter.string(from: self)
     }
 
-    var shortYear: String {
-        return formatWith(dateFormat: "yy")
+    var formattedRFC822Date: String {
+        return DateFormatter.rfc822DateFormatter.string(from: self)
     }
 
-    func formattedSpanishDate(timeZone: TimeZone? = nil) -> String {
-        return formatWith(dateFormat: "dd 'de' MMMM", timeZone: timeZone)
+    var formattedSpanishDay: String {
```

Comment:
```suggestion
    var formattedDay: String {
```
Spanish is not relevant here